recipes/mykrobe/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: mykrobe
+  version: "0.5.6"
+
+source:
+  fn: mykrobe-0.5.6.tar.gz
+  url: https://pypi.python.org/packages/7e/35/a662ec560cfc91f35519e1459de917001ca19a9dbefd53c5b3a2b14122d0/mykrobe-0.5.6.tar.gz
+  md5: 5e912497f98cc192e3f0fbf426210560
+
+build:
+  number: 0
```

Comment:
Please add the following to the `build:` section to get entry points created by Conda:
```yaml
  entry_points:
    - mykrobe = mykrobe.mykrobe_predictor:main
```

Reply:
Added